core/kazoo_ast/src/kz_ast_util.erl

```diff
@@ -482,9 +482,12 @@ cell_wrap('undefined') -> <<" ">>;
 cell_wrap([]) -> <<"`[]`">>;
 cell_wrap(L) when is_list(L) -> [<<"`[\"">>, kz_binary:join(L, <<"\", \"">>), <<"\"]`">>];
 cell_wrap(<<>>) -> <<"\"\"">>;
-cell_wrap(?EMPTY_JSON_OBJECT) -> <<"`{}`">>;
+cell_wrap(B) when is_binary(B) -> [<<"`">>, B, <<"`">>];
 cell_wrap(Type) ->
-    [<<"`">>, kz_term:to_binary(Type), <<"`">>].
+    case kz_json:is_json_term(Type) of
```

Comment:
Out of curiosity, what's triggering this change to be necessary?

Reply:
We have set `properties.gateways.custom_sip_headers.default` in the resources schema to be:

    {
        "X-Account-ID": "{account_id}",
        "X-Billing-Number": "{caller_id_number}"
    }

This crashes kz_ast_util while trying to build the table.